BlazarUI/app/scripts/utils/user.js

```diff
@@ -0,0 +1,3 @@
+import store from 'store';
+
+export const isBetaUser = store.get('blazar-beta');
```

Comment:
@jonathanwgoodwin this is where the current check is. we'll need to inject something here